src/pivotal-ui/components/links/links.scss

```diff
@@ -35,6 +35,7 @@ Link
 <a class="link-text" href="http://bit.ly/1ulOAW7" target="_blank">default link</a>                                | `link-text`                     | Important links
 <a class="link-text link-lowlight" href="http://bit.ly/1ulOAW7" target="_blank">lowlight link</a>                 | `link-text link-lowlight`       | Less important links
 <a class="link-text link-inverse bg-dark-1" href="http://bit.ly/1ulOAW7" target="_blank">inverse link</a>         | `link-text link-inverse`        | I go on a non-white background
+<a class="link-text link-lowlight link-hover" href=http://bit.ly/1ulOAW7" target="_blank">hover highlight link</a>      | `link-text link-lowlight link-hover`        | Links that change color on hover
```

Comment:
this class name is kind of specific to how it happens to look visually (which, alas, often changes)